cdc/log.cc

```diff
@@ -835,8 +835,14 @@ class transformer final {
         auto&& cc = _schema->clustering_key_columns();
 
         std::vector<query::clustering_range> bounds;
-        if (cc.empty()) {
+        uint32_t row_limit = query::max_rows;
```

Comment:
What's the difference between query::max_rows and query::max_partitions that was used before?

Reply:
There is no difference - the values of `query::max_rows` and `query::max_partition` are the same and have the same type. I changed it because the fourth argument to `read_command`'s constructor takes a limit of _rows_ to be returned, not the limit of _partitions_ to be returned.

Reply:
ok - thanks for the explanation.